packages/gatsby-theme-newrelic/src/utils/page-tracking/tessen.js

```diff
@@ -26,29 +27,58 @@ const trackViaTessen = ({ location }, themeOptions) => {
     return;
   }
 
-  const { pageView } = tessenConfig;
-  const tessen = createTessen(tessenConfig);
+  window.initializeTessenTracking = initializeTessenTracking(
+    tessenConfig,
+    env,
+    location
+  );
 
-  if (!canSendPageView(pageView)) {
-    return warnAboutNoop(pageView);
-  }
+  window.initializeTessenTracking();
+  // wrap inside a timeout to make sure react-helmet is done with its changes (https://github.com/gatsbyjs/gatsby/issues/11592)
```

Comment:
```suggestion
  window.initializeTessenTracking();
  
  // wrap inside a timeout to make sure react-helmet is done with its changes (https://github.com/gatsbyjs/gatsby/issues/11592)
```

[nit] Just adding some whitespace for clarity